src/FormGroup/FormGroup.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import FormGroup, { FormGroupProps } from './index.js'
+import InputText from '../Input/index.js'
+
+export default {
+  title: 'Components/FormGroup',
+  component: FormGroup,
+  argTypes: {
+    onSubmit: {},
+  },
+}
+
+const DefaultStoryTemplate = (args: FormGroupProps) => (
+  <FormGroup onSubmit={args.onSubmit}>
```

Comment:
could use the `action` function from Storybook https://github.com/digicatapult/ui-component-library/blob/bd84e0a6eaa896b86944d509abd54dd33ff9c522/src/Search/Search.stories.tsx#L31

Reply:
then can show in the storybook actions tab when you submit e.g. (from Search component) 
<img width="207" alt="image" src="https://github.com/digicatapult/ui-component-library/assets/40722025/a17c7d7f-f385-4e06-bc9e-3a840f3dd472">

Reply:
instead of a window alert

Reply:
thanks!